bugbug/model.py

```diff
@@ -24,8 +25,8 @@ def __init__(self, lemmatization=False):
         else:
             self.text_vectorizer = TfidfVectorizer
 
-        self.undersampling_enabled = True
         self.cross_validation_enabled = True
+        self.sampler = RandomUnderSampler(random_state=0) if self.undersampling_enabled else self.sampler
```

Comment:
No, make this None by default, and simply define it in each model as needed.
Remove the self.undersampling_enabled attribute.